hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/ImporterIntegrationTest.java

```diff
@@ -126,6 +126,11 @@ protected <T> Collection<T> findHistory(Class<T> historyClass, String ids) {
         return findHistory(historyClass, ids, null);
     }
 
+    protected <T> Collection<T> findHistory(String table, Class<T> historyClass) {
```

Comment:
An overloaded method should keep the parameters in the same order.

```suggestion
    protected <T> Collection<T> findHistory(Class<T> historyClass, String table) {
```

Reply:
There is already `protected <T> Collection<T> findHistory(Class<T> historyClass, String ids)` though, and that makes the compiler unhappy. Looks like I went for the simple solution. 😄 

If that is not acceptable, the new method is only called twice from `UpdateTokenFreezeKycStatusMigrationTest`. I can see the default ID column it's computing for the `PostMigrationToken` "entity" is `token_id` based on the `@Id` annotation on the `tokenId` field.

So, I could call the existing `protected <T> Collection<T> findHistory(Class<T> historyClass, String ids, String table)` knowing that information.

Reply:
Ok, I didn't see the existing one since it's not in the diff. Can keep.